### PR TITLE
SendTxProcess: Fix style issues

### PR DESCRIPTION
**Pull request by Geod24**
Trivial fix, had the commit in a local branch for months, figured I should get rid of it 🤷

### CODE REVIEW

**Review by hewison-chris — APPROVED** [review]
LGTM

**Review by linked0 — APPROVED** [review]

File: source/agora/cli/client/SendTxProcess.d [hunk 1]
```diff
@@ -124,11 +124,12 @@ public void printSendTxHelp (ref string[] outputs)
 
     Params:
         args = client command line arguments
+        outputs = Where to output any send output (1 line will be one entry)
```
Comment (linked0) [inline source/agora/cli/client/SendTxProcess.d]: Isn't it to be `any sent output`?

Comment (Geod24) [reply source/agora/cli/client/SendTxProcess.d]: Reading again, that doc is terrible. Will fix.

Comment (Geod24) [reply source/agora/cli/client/SendTxProcess.d]: Done.

**Review by kchulj — APPROVED** [review]
LGTM

**Pull request merged**